device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/IotHubEventCallback.java

```diff
@@ -1,22 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-package com.microsoft.azure.sdk.iot.device;
-
-/**
- * An interface for an IoT Hub event callback.
- *
- * Developers are expected to create an implementation of this interface,
- * and the transport will call {@link IotHubEventCallback#execute(IotHubStatusCode, Object)}
- * upon receiving a response from an IoT Hub.
- */
-public interface IotHubEventCallback
```

Comment:
Replaced with MessageSentCallback, MessagesSentCallback, and SubscriptionAcknowledgedCallback